src/pages/ListPage/PolicyTableError.ts

```diff
@@ -0,0 +1,43 @@
+import { ErrorContentProps } from '../../components/Policy/PolicyTable';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+
+export const policyTableError = (canReadAll: boolean, requestHasError?: boolean, httpCode?: number): ErrorContentProps | undefined => {
+    if (!canReadAll) {
+        return {
+            icon: ExclamationCircleIcon,
+            title: 'No permission to view this page',
+            content: 'You do not have permission to view this page'
+        };
+    }
+
+    if (requestHasError) {
+        switch (httpCode) {
+            case 404:
```

Comment:
I think this should explicitly test for 403 as well (basically the `!canReadAll` case)

Reply:
> I think this should explicitly test for 403 as well (basically the `!canReadAll` case)

done